x-pack/solutions/observability/plugins/apm/server/plugin.ts

```diff
@@ -258,20 +258,17 @@ export class APMPlugin
 
     // create .apm-agent-configuration index without blocking start lifecycle
     createApmAgentConfigurationIndex({ client, logger }).catch((e) => {
-      logger.error('Failed to create .apm-agent-configuration index');
-      logger.error(e);
+      logger.debug(`Failed to create .apm-agent-configuration index: ${e.message}`);
     });
 
     // create .apm-custom-link index without blocking start lifecycle
     createApmCustomLinkIndex({ client, logger }).catch((e) => {
-      logger.error('Failed to create .apm-custom-link index');
-      logger.error(e);
+      logger.debug(`Failed to create .apm-custom-link index: ${e.message}`);
```

Comment:
@kpatticha We have a similar logic in the observability plugin. ([code](https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/observability/server/utils/create_or_update_index.ts#L63))
What is this failure exactly about?